src/modules/esl-forms/esl-select/core/esl-select-dropdown.ts

```diff
@@ -93,14 +94,10 @@ export class ESLSelectDropdown extends ESLToggleable {
   }
 
   @bind
-  public updatePosition(): void {
-    if (!this.activator) return;
-    const windowY = window.scrollY || window.pageYOffset;
-    const rect = this.activator.getBoundingClientRect();
-
-    this.style.top = `${windowY + rect.top + rect.height}px`;
-    this.style.left = `${rect.left}px`;
-    this.style.width = `${rect.width}px`;
+  public updateWidth(): void {
```

Comment:
Should be truthful part of `esl-popup` recalculations as a temporary solution consider `_updatePosition` extension (independently of reflow issue, just check the visual part of change)